source/clear-linux/guides/tooling/swupd-guide.rst

```diff
@@ -118,17 +114,18 @@ The data that an update provisions to a system can be obtained in
 three different ways. There are three different methods, and they
 exist to optimize the delivery of content and speed up updates.
 
```

Comment:
Suggest replacing sentences above with this:
To speed up updates and optimize content delivery, update data provisioned to a system is obtained by one of the following methods:

Reply:
I like your version much better. LGTM.